### PR TITLE
B9lab documentation updates - package 2

### PR DESCRIPTION
**Pull request by coldice**
* Restructures the whole docs section, removes almost all archived and unlinked pages.
* Updates the Contributor Guide and DApp Integration Center sections.
* Requires language translations (1 string), english string provided in ko, ar, ru and zh .yml language file.

We have removed the /contribute section and links to it, and moved most of its content to /our_team (including the jobs section). **You might want to add new links to it to the main nav, sidebar and footer**, the section is currently not linked.

There have also been a few removals on the desktop content from your side, which we merged in. Please let me know if you would like to discuss any content details.

A quick note:
The /get/ landingpage contains links to /CHANGE_ME - this is also on dev.status.im, so I think you are currently changing this page?

Please let me know if you would like to discuss further content details separately.

### CODE REVIEW

File: source/_data/sidebar.yml [hunk 1]
```diff
@@ -33,189 +38,15 @@ docs:
...
-        path: status_go.html
-      - title: Your First PR
-        path: first_pr_guide.html
-      - title: A Makefile Tour
-        path: makefile_tour.html
-      - title: Status-React Quick Start
-        path: status_react_quickstart.html
-
-
-  - title: Tutorials
```
Comment (jakubgs) [inline source/_data/sidebar.yml]: Why are we removing this whole section?

File: source/_data/sidebar.yml [hunk 2]
```diff
@@ -33,189 +38,15 @@ docs:
...
-          - title: SET PINLESS PATH
-            path: setpinlesspath.html 
-          - title: EXPORT KEY
-            path: exportkey.html
-          - title: STORE DATA
-            path: storedata.html
-          - title: GET DATA
-            path: getdata.html
-
-  - title: Research
```
Comment (jakubgs) [inline source/_data/sidebar.yml]: Why are we removing this whole section?

File: source/_data/sidebar.yml [hunk 3]
```diff
@@ -33,189 +38,15 @@ docs:
...
-      - title: Nimbus
-        path: https://nimbus.team/
-      - title: Whisper In Your CLI
-        path: ../tutorials/whisper_basic_cli.html
-      - title: Whisper with Embark
-        path: ../tutorials/whisper_embark.html
-      - title: Status Analytics
-        path: https://analytics.status.im/
-
-  - title: Security
```
Comment (jakubgs) [inline source/_data/sidebar.yml]: Why are we removing this whole section?

File: themes/navy/languages/ko.yml [hunk 4]
```diff
@@ -360,7 +360,7 @@ site:
         description: '스테이터스 <br> 개발을 시작하기 위해 필요한 모든 것'
         link: '개발 문서'
         link-1: 시작하기
-        link-2: '스테이터스 빌드하기'
+        link-2: Contributor Guide
```
Comment (jakubgs) [inline themes/navy/languages/ko.yml]: Why are you changing the translations her?

File: themes/navy/layout/security.ejs [hunk 5]
```diff
@@ -1,36 +1,36 @@
...
-		</div>
+    <div class="container m-b-100">
+      <div class="row">
+        <div class="col-lg-8">
+          <h1><%- __('site.security.intro.title') %></h1>
+          <p class="teaser-1"><%- __('site.security.intro.description-1') %></p>
+          <p class="teaser-1"><%- __('site.security.intro.description-2') %></p>
+        </div>
+      </div>
+    </div>
```
Comment (jakubgs) [inline themes/navy/layout/security.ejs]: Don't mix tabs and spaces.

**Comment by jakubgs** [comment]
Security page formatting is broken:
security_page_formatting_broken

File: themes/navy/languages/ko.yml (hunk 4 above)
Comment (coldice) [reply themes/navy/languages/ko.yml]: The linked item changed, this was "Build Status" before - needs a new translation.

File: themes/navy/layout/security.ejs (hunk 5 above)
Comment (coldice) [reply themes/navy/layout/security.ejs]: Will fix. Looks like this is inconsistent across a few files (eg. https://github.com/status-im/status.im/blob/develop/themes/navy/layout/index.ejs#L9 and https://github.com/status-im/status.im/blob/develop/themes/navy/layout/get.ejs#L7) - should we stick to tabs?

**Comment by coldice** [comment]
The removal of content sections and links has been discussed by Rob and Jonathan, but probably not all implications were visible (like the keycard API being linked from keycard.tech). I will ping Rob to recheck with Jonathan.

**Comment by jakubgs** [comment]
Please don't delete the `source/docs/FAQs.md` link.

If Rob and Jonathan requested these changes then they should review them here on GH.

**Comment by coldice** [comment]
@jakubgs I am checking with them on the removal, I will add another commit to undo the removal of the FAQ and probably the keycard section, waiting for a reply.

For documentation purposes (sorry I forgot to add this to the initial PR message), we have removed the following:
Sections:
- REM source/audits/*
- REM source/guides/*
- ~REM source/keycard_api/*~
- REM source/org_sec/*
- ~REM source/docs/FAQs.md~
- REM source/contribute/*
- REM source/security_docs/*
- REM source/tutorials/*
- REM source/technical/FAQ-desktop.md
- CREATE source/our_team - move sections from /contribute, /org_sec and /guides in

Links:
- REM links to /contribute/open_positions.html and /contribute/life_status.html from about page
- REM link to /get-involved/developers/ from themes/navy/layout/community.ejs

**Comment by j-zerah** [comment]
1. @jakubgs - i thought we had moved the keycard docs off of the Status.im domain and onto the Keycard domain? There is an email thread with Tim Hartman in which you confirmed this. However, it is clearly not deployed. https://dev.keycard.tech/docs/

Please remove keycard docs from status.im docs

2. We still need the FAQs in the docs for now as we do link to it from the site. We can find a better place for it in the future but for now, please keep it.

3. Removed section "Tutorials" - yes please remove these. They make no sense currenrtly

4. Removed section "Research" - yes please remove. This is not documentation and we have decided to remove

5. Removed section "Security" - yes we have updated https://status.im/security/ and also created a new security repo with all this info

**Comment by jakubgs** [comment]
Okay. I see what happened. The site repo was updated but the changes were never deployed. So I fixed the deployments in https://github.com/status-im/keycard-site/commit/f18ddae765b1b5c898059af3c8a17f43c63e68a2 and also had to fix the Docs URLs in few places in https://github.com/status-im/keycard-site/commit/20fa22df1d27de6b5bdf6aef3305f9ec2d0fbd2c so the Keycard API docs should be in the right place now.

**Comment by coldice** [comment]
Thanks for the confirmation, PR is updated: Link is back

**Comment by coldice** [comment]
@jakubgs perfect, let's merge 👍

**Comment by jakubgs** [comment]
Deployed with https://ci.status.im/job/misc/job/dev.status.im/851/. Looks good:
* https://dev.status.im/docs/
* https://dev.status.im/security/

For now these changes will stay in `dev` environment until V1 is out.

**Comment by jakubgs** [comment]
Nice, this PR fucked up a bunch of our pages, including an important one at:
https://status.im/contribute/open_positions.html
Screenshot_2020-02-14_at_10 47 01
I blame you @j-zerah. But now the question is, how do we want to recover this?

A bunch of different things were thrown into `our_team` folder but never linked from anywhere. What am I supposed to do with those pages?

**Comment by coldice** [comment]
Hey @jakubgs, I mentioned this move on the PR message, as I noticed updates were done to the jobs and contribute pages while we were moving the things, but this area was out of scope for us.
We did also disable the link the footer which you might want to adjust here: https://github.com/status-im/status.im/blob/develop/themes/navy/layout/partial/footer.ejs#L40

The list above (https://github.com/status-im/status.im/pull/455#issuecomment-581977783) shows further sections moved to our team. We did collect them in this new folder for you, but did not link them or add an index page, as it was out of scope and also probably something you want to discuss and decide on further. I feel this piece of information got a bit lost on the way.

**Pull request merged**